typeql/language/insert.feature

```diff
@@ -2772,3 +2770,33 @@ Parker";
       $x isa bird;
       $x iid V123;
       """
+
```

Comment:
We seem to have 2 lines separating scenarios. Could you check and follow whatever the convention is?